Views/GICommitSplitterViewController.m

```diff
@@ -87,7 +87,7 @@ - (void)viewWillShow {
   XLOG_DEBUG_CHECK(_commit);
   [super viewWillShow];
   
-  _titleTextField.stringValue = [NSString stringWithFormat:@"\"%@\" <%@>", _commit.summary, _commit.shortSHA1];
```

Comment:
This should be changed into NSLocalizedString() in a separate and prior commit to make it clear it's a UI string.